### PR TITLE
Handle song get & post request

### PR DESCRIPTION
**Pull request by khanhphd-student**

### CODE REVIEW

File: server/controller/Songs/attribute.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
...
+module.exports = {
     songID: `songID`,
     songName: `songName`,
     uploadDate: `uploadDate`,
     numberOfLikes: `numberOfLikes`,
-    duration: `duration`
+    duration: `duration`,
+    genre: `genre`,
+    artist: `artist`,
+    likeBy: `likeBy`
```
Comment (TrNgTien) [inline server/controller/Songs/attribute.js]: delete this please likeBy: `likeBy`

File: server/controller/Songs/songQueries.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  postSongBasicInfo: (data, callBack) => {
+    dbConfig.query(
+      `INSERT INTO songs (songName, uploadDate, numberOfLikes, duration) VALUES (?,?,?,?)`,
+      [data.songName, data.uploadDate, data.numberOfLikes, data.duration],
+      (err, results, fields) => queryCallBackHandle(err, results, fields)
+    );
+  },
+  postSongArtistInfo: (data, callBack) => {
+    dbConfig.query(
+      `INSERT INTO artists (artistID,artistName) VALUES (?,?)`,
```
Comment (TrNgTien) [inline server/controller/Songs/songQueries.js:43]: please make the artistID is auto increase ! @Dev01-vinamr-Ken

File: server/controller/Songs/songQueries.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      [data.artistID, data.artistName],
+      (err, results, fields) =>
+        queryCallBackHandle(err, results, fields, callBack)
+    );
+  },
+  postSongGenreInfo: (data, callBack) => {
+    dbConfig.query(
+      `INSERT INTO genres (genreID,genreName) VALUES (?,?)`,
```
Comment (TrNgTien) [inline server/controller/Songs/songQueries.js:51]: please make the genreID is auto increase ! @Dev01-vinamr-Ken

File: server/controller/Songs/songHandler.js [hunk 4]
```diff
@@ -1,39 +1,97 @@
...
+
+    let getSongIdQueryResult = await sqlQuery(connection, getSongIdQuery);
+    let getArtistIdQueryResult = await sqlQuery(connection, getArtistIdQuery);
+    let getGenreIdQueryResult = await sqlQuery(connection, getGenreIdQuery);
+
+    let newestSongId = getSongIdQueryResult[0].songID + 1;
+    let newestArtistId = getArtistIdQueryResult[0].artistID + 1;
+    let newestGenreId = getGenreIdQueryResult[0].genreID + 1;
+
+    let postSongQueryBasic = `INSERT INTO songs (songID, songName, uploadDate, numberOfLikes, duration) VALUES (${newestSongId},?,?,?)`;
```
Comment (TrNgTien) [inline server/controller/Songs/songHandler.js]: Please delete auto generate for songID, it has already had! @Dev01-vinamr-Ken

File: server/database/db_connection.js [hunk 5]
```diff
@@ -1,21 +1,22 @@
...
-  
-          return;
-        }
-        resolve(connection);
-      });
+  user: "b4bb330a3f009b",
+  host: "eu-cdbr-west-01.cleardb.com",
+  password: "4be56a74",
+  database: "heroku_1fcc54407741c0e",
+  multipleStatements: true
```
Comment (TrNgTien) [inline server/database/db_connection.js]: What's this, please delete that

**Review by TrNgTien — APPROVED** [review]

**Review by nguyenquocviet2007 — APPROVED** [review]

**Pull request merged**